DIY_Thermal_Light_Painting/DIY_Thermal_Light_Painting.py

```diff
@@ -0,0 +1,75 @@
+#  Designed specifically to work with the MLX90614 sensors in the
+#  adafruit shop
+#  ----> https://www.adafruit.com/products/1748
+#  ----> https://www.adafruit.com/products/1749
+#
+#  These sensors use I2C to communicate, 2 pins are required to
+#  interface Adafruit invests time and resources providing this open
+#  source code,
+#  please support Adafruit and open-source hardware by purchasing
+#  products from Adafruit!
+
+import time
+import board
+import busio as io
+import neopixel
+import adafruit_mlx90614
+
+# use bitbangio only with ESP8266
+# * does not support hardware i2c
+# * comment out busio above
+# * express boards can use also bitbangio, but they have i2c hardware built-in
+#import bitbangio as io
```

Comment:
This isn't needed. We'll use bitbangio internally on the ESP8266 automatically now.

Reply:
ooh i didnt know that, we'll clean that up on some old guides!